android/hello_sdl_android/src/main/java/com/sdl/hellosdlandroid/SdlReceiver.java

```diff
@@ -41,9 +41,7 @@ public void onReceive(Context context, Intent intent) {
 			String action = intent.getAction();
```

Comment:
The whole if statement from line 40 to 49 can be removed.  Also, any unused vars and imports (as a result of the ode removal) can be removed as well.